QUEST.tsv

```diff
@@ -2384,7 +2384,7 @@ QUEST_20150803_002390	The death moves away from me whenever I set Owl Sculptures
 QUEST_20150803_002391	Are you interested in the eternal life?{nl}We, Necromancers control the deads so it's easy when you think about that.{nl}
 QUEST_20150803_002392	Absorbing the vitality and extend the life is possible for me.{nl}But, it's not possible for humans.
 QUEST_20150803_002393	You want to know how I live an eternal life.{nl}We could say stopping the time can be also eternal.{nl}
-QUEST_20150803_002394	I am maintaining the body that doesn't get old nor die by stopping the time of the body.{nl}I don't know if I could say stopping the time is the same as being eternal.
+QUEST_20150803_002394	I am maintaining the body that doesn't get old or die by stopping the body's biological clock.{nl}I don't know if I could say stopping the time is the same as being eternal.
```

Comment:
It would appear that the person in question here is talking about "aging".  The rest of the line could use a little work as well.  Also nor is technically correct here ( you could use "or" also, but since this is a negative "nor" works too ).  I would suggest:
"I am maintaining the body that doesn't get old nor die by stopping the aging process.{nl}I don't know if I could say stopping this process is the same as being eternal."
Also, I don't mean to nitpick, but the surrounding lines could use some work as well.  Would it be possible to look over more than one line for a pull request?
Ex: 
line 2391 "control the deads"
line 2392 "Absorbing the vitality and extend the life"